build-tests.cmd

```diff
@@ -0,0 +1,24 @@
+@echo off
+setlocal
+
+set buildTests=build-tests.log
+echo Running build-tests.cmd %* > %buildTests%
+
+if /I [%1] == [/?] goto Usage
+if /I [%1] == [/help] goto Usage
+
+echo msbuild.exe %~dp0src\tests.builds %* /nologo /maxcpucount /v:minimal /clp:Summary /nodeReuse:false /flp:v=detailed;Append;LogFile=%buildTests% >> %buildTests%
+call msbuild.exe %~dp0src\tests.builds %* /nologo /maxcpucount /v:minimal /clp:Summary /nodeReuse:false /flp:v=detailed;Append;LogFile=%buildTests%
```

Comment:
Yes, I created an issue to add this capability to all the dev workflows scripts, as we had it in our internal dev workflow.
Issue #7329